### ISSUE TITLE
Using different dataset for Email spam detection model

### ISSUE THREAD

**Issue by Hritaku** [OpenCodeEra/ML-X#2]
### Hey 👋, I am a Hacktoberfest'23 contributor 

I would like to use  different dataset for the email spam detection model

## Code of Conduct
I agree to follow this project's Code of Conduct

> **Kindly assign it to me under _HacktoberFest'23_** 
> **I will surely complete it ASAP**

**Comment by AKACHI-4** [comment]
Hey @Hritaku, I'm glad you're interested in contributing! Could you please provide some details about the dataset ? kindly mention all that in issue...

**Comment by Hritaku** [comment]
@AKACHI-4 sure
This is the [Dataset](https://www.kaggle.com/datasets/balaka18/email-spam-classification-dataset-csv) I want to add.

**Comment by AKACHI-4** [comment]
Yeah great @Hritaku ... go ahead.. but don't change in existing notebook for that .. kindly add a new notebook u can give it name `notebook-1` while commiting... 

also don't forget to take notes so other can understand it ..

**Comment by QQ07** [comment]
Can I do it?

**Comment by AdarshRawat1** [comment]
@QQ07 Yes, you can make a PR and mention this issue.
